Documentation/how-to-build-and-run-ilcompiler-in-visual-studio-2015.md

```diff
@@ -12,7 +12,7 @@ _Note_:
   - Set "desktop" project in solution explorer as your startup project
 
   - Set startup command line to:
-`c:\corert\src\ILCompiler\repro\bin\Debug\repro.exe -r c:\corert\bin\Product\Windows_NT.x64.Debug\System.Private.CoreLib.dll -r <repo_root>\bin\tests\package\toolchain.<OS>-<Arch>.Microsoft.DotNet.App.1.0.0-prerelease\*.dll -out c:\corert\src\ILCompiler\reproNative\repro.obj`
+`c:\corert\src\ILCompiler\repro\bin\Debug\repro.exe -r c:\corert\bin\Product\Windows_NT.x64.Debug\System.Private.CoreLib.dll -r <repo_root>\bin\tests\package\toolchain.<OS>-<Arch>.Microsoft.DotNet.AppDep.1.0.0-prerelease\*.dll -out c:\corert\src\ILCompiler\reproNative\repro.obj`
```

Comment:
This should now be AppDep.1.0.1.

I was sitting next to so I know you updated it, but you probably didn't hit save.